### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by Kakalanp**
## New files:
- App.css:
  - Adds “general styling for the page” (font-family).
- Home.js:
  - The landing page, it only contains a title and Lorem text.
- Home.module.css:
  - Centers the tittle.
  - Adds a little padding.
- Quote.js:
  - Displays a quote from William Thurston.
- Quote.module.css:
  - Centers the above-mentioned quote.
- Navbar.js:
  - Displays a list of the pages in this project.
  - Redirects the user to said pages on click.
- Navbar.module.css:
  - Styles items to look decent.
  - Organizes items in the navigation bar.

## Updated files:
- index.js:
  - Added [React Router](https://reactrouter.com/)
  - Added Navigation bar for the page.
- App.js:
  - Added Routes for the different pages.
- Calculator.js:
  - Added remaining page contents.
- Calculator.module.css:
  - Restricted the calculator to only a portion of the screen.
  - Adding a black border to the calculator.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+
+import './App.css';
 
 export default function App() {
   return (
-    <Calculator />
+    <Routes>
+      <Route path="Math-Magicians/Calculator" element={<Calculator />} />
+      <Route path="Math-Magicians/" element={<Home />} />
+      <Route path="Math-Magicians/Quote" element={<Quote />} />
```
Comment (FrederickMih) [inline src/App.js:15]: - [ ] Hi @Kakalanp, great job on this project so far :+1: however, the navigation links for all your three routes are not working properly. When I click on a link, the content is not displayed. Please fix this issue and submit your project once more.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Kakalanp , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Kakalanp) [reply src/App.js:15]: Looks like I messed up that part in my last commit, trying to match the one from the deployment page. 🧐 

I forgot to correctly add 'Math-Magicians/' part before the actual path in the nav-links: 😅 https://github.com/Kakalanp/Math-Magicians/blob/65018dfb34c2fd2096b8916a2cde4d88b8101f03/src/components/Navbar.js#L27

Fixing it right now! 💯 thanks.

**Review by rotimiazeez — APPROVED** [review]
# Hi @Kakalanp 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
> # Hi @Kakalanp 👋,
> ### Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks, Merging right now! 👍

**Pull request merged**